### ISSUE TITLE
Bower needs package json

### ISSUE THREAD

**Issue by tbba** [danielemoraschi/maplace.js#71]
Bower needs a proper "package.json" file as part of the component
or it does not work correctly - at least not with the software I use...

**Comment by tbba** [comment]
Hm... I wished I could help more. I just can confirm that the maplace bower package does not work correctly with the bower software I use (CodeKit/Mac). The above (my first msg) was a quote from the CodeKit support). Maybe that was wrong, and I did not understand your reply. I am only bower user, not a coder :-)

What basically happens is that maplace.js gets downloaded but isn't recognized thereafter.

Not sure it helps, but I compared your json file with the others, that work, and found one difference:
They don't but a "./" before the "main" path.
so:
"main": "src/maplace-0.1.3.min.js",
instead of:
"main": "./src/maplace-0.1.3.min.js",

Not sure that this solves the problem (on a Mac) but maybe that is worth trying?

I really would love to have maplace.js working with bower some day. :-)

**Comment by tbba** [comment]
or is this maybe a version mismatch somewhere?
some info says 0.1.3 and some other 0.1.32

**Comment by tbba** [comment]
This is a comment from the author of CodeKit:
bower.json is a file that tells Bower what to install when you run bower on that component’s project. It’s how you quickly install dependencies, etc. A “package.json” it what Bower looks at for information about the current component. If one is missing, that component will not show up as installed because Bower doesn’t see it when it scans for installed components. Your component author should have a package.json file in his component.

**Comment by danielemoraschi** [comment]
Please try now:
```
bower info maplace-js
```

**Comment by tbba** [comment]
Thanks much for looking into this. 
There is no change here. The software is missing a file called "package.json" and lists the plugin status  as "unknown" - so updates get ignored - what is not what Bower is intended for.
CodeKit is a very popular Mac Software for handling Bower files and I am a user, not the coder :-)

**Comment by danielemoraschi** [comment]
package.json is for npm not bower.
If you try
```
bower install maplace-js
```
You get:
```
Danieles-MBP:test dmoraschi$ bower install maplace-js
bower maplace-js#*              cached git://github.com/danielemoraschi/maplace.js.git#0.1.33
bower maplace-js#*            validate 0.1.33 against git://github.com/danielemoraschi/maplace.js.git#*
bower maplace-js#*                 new version for git://github.com/danielemoraschi/maplace.js.git#*
bower maplace-js#*             resolve git://github.com/danielemoraschi/maplace.js.git#*
bower maplace-js#*            download https://github.com/danielemoraschi/maplace.js/archive/v0.1.33.tar.gz
bower maplace-js#*             extract archive.tar.gz
bower maplace-js#*            resolved git://github.com/danielemoraschi/maplace.js.git#0.1.33
bower jquery#~1.9.0             cached git://github.com/jquery/jquery.git#1.9.1
bower jquery#~1.9.0           validate 1.9.1 against git://github.com/jquery/jquery.git#~1.9.0
bower maplace-js#~0.1.33       install maplace-js#0.1.33
bower jquery#~1.9.0            install jquery#1.9.1

maplace-js#0.1.33 bower_components/maplace-js
└── jquery#1.9.1
```
Here the content of the auto-generated bower folder:
```
Danieles-MBP:test dmoraschi$ ls -lh bower_components/
drwxr-xr-x  25 dmoraschi  staff   850B 31 Jan 18:29 jquery
drwxr-xr-x  15 dmoraschi  staff   510B 31 Jan 18:29 maplace-js
```

**Comment by tbba** [comment]
There is another issue with automatic updates through bower:
if you name the file
src/maplace-0.1.3.min.js
and this one is updated and has a new number, you have a new file and not overwritten the old one.
If my software code linked to the maplace-0.1.3.min.js, after a bower update, I still link to the old file (or get a file not found error - depending on how I do the update).

**Comment by tbba** [comment]
Hi, there is no input field inside of CodeKit to set commands like "bower install maplace-js". :-)

>package.json is for npm not bower.
Well, I do not know what "belongs" to what. That is not my point.

I only see that most other bower installs at GitHub I know include a package.json and work flawlessly - and I like maplace a lot and would wish it would work with Bower/CodeKit too - so I have all my plugins updated automatically - which is what Bower is for in the first place.

**Comment by danielemoraschi** [comment]
Never used CodeKit, and I'm sorry to say that but I can support bower or npm or any other package manager for development as required but not commercial software.
However, I'm trying now the trial and I've installed successfully Maplace with not problem at all, getting the latest version of the code. 

The file name **must be** maplace-0.1.3.min.js for the latest version.

I've added the package.json file.

The point is that I was trying to solve a problem about Bower that actually was not about Bower..

Hope this will solve your problem!
